app/views/collection_items/_collection_item_form.html.erb

```diff
@@ -2,7 +2,11 @@
 <% @collection_item ||= CollectionItem.new %>
 <% in_page ||= false %>
 <div id="collection-form">
-  <h3 class="heading"><%= ts("Add %{title} to collections", :title => item.respond_to?(:title) ? item.title : item.class.name).html_safe %></h3>
+  <% if item.is_a?(Work) %>
+    <h3 class="heading"><%= t(".invite_header", title: item.title).html_safe %></h3>
+  <% else %>
+    <h3 class="heading"><%= t(".add_header", title: item.class.name) %></h3>
```

Comment:
I think the Translation Committee [doesn't generally like passing class names](https://otwarchive.atlassian.net/browse/AO3-5867) like "Bookmark" as interpolated variables. But since this is only for bookmarks at the moment, you can probably just specialize `.add_header` to be for bookmarks, instead of having `".add_header_#{item.model_name.i18n_key}"` as the translation key.